src/miroil/gesture_ender.cpp

```diff
@@ -0,0 +1,68 @@
+/*
+* Copyright © Canonical Ltd.
+ *
+ * This program is free software: you can redistribute it and/or modify it
+ * under the terms of the GNU General Public License version 2 or 3 as
+ * published by the Free Software Foundation.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#include "gesture_ender.h"
+
+#include <mir/frontend/pointer_input_dispatcher.h>
+
+namespace
+{
+struct PointerInputDispatcher : mir::frontend::PointerInputDispatcher
```

Comment:
```suggestion
struct GestureEnder : mir::frontend::PointerInputDispatcher
```

Or something along those lines?